c2corg_ui/templates/helpers/view.html

```diff
@@ -6,6 +6,61 @@
     from c2corg_ui.views.document import get_slug
 %>
 
+<%def name="get_licence(doc)">\
+<%
+  doctype = doc['type']
+  CC_ND_NC = False
+  CC_BY_SA = False
+
+  if doctype == 'o' or doctype == 'profile' or doctype == 'xreport':
```

Comment:
Do we really have doctypes ``profile`` or ``xreport``? That does look consistent with the 1-letter doctypes!